etc/di.xml

```diff
@@ -28,4 +28,16 @@
             </argument>
         </arguments>
     </type>
+    <virtualType name="customStagedProductAttributeSubscription" type="Aligent\PrerenderIo\Model\Mview\View\Attribute\Subscription">
```

Comment:
This virtual type name should be more specific, also.

Additionally, you don't actually need a virtual type here - you can just define the `entityInterface` argument for the type itself (and then use the type as an argument for `SubscriptionFactory`